src/plugins/gauge/components/GaugeRadial.vue

```diff
@@ -0,0 +1,441 @@
+            </div>
+
+            <div class="c-dial">
+                Dial Clip
+                <div class="c-dial__value"
+                     style="clip-path: polygon(0 0, 100% 0, 100% 100%, 50% 50%, 0 100%);"
+                ></div>
+            </div>
+
+            <div class="c-dial">
+                Dial Clip 180
+                <div class="c-dial__value c-dial-clip--180"></div>
+            </div>
+
+            <div class="c-dial">
+                Dial Clip 90
+                <div class="c-dial__value c-dial-clip--90"></div>
+            </div>
+        </div>
+
+        <div v-if="limitLow && limitHigh && debugDial"
+             class="temp-dial-debug"
+        >
+            <div class="c-dial">
+                High Limit
+                <svg class="c-dial__bg"
+                     viewBox="0 0 512 512"
+                >
+                    <path d="M100,256A156,156,0,1,1,366.3,366.3L437,437a255.2,255.2,0,0,0,75-181C512,114.6,397.4,0,256,0S0,114.6,0,256A255.2,255.2,0,0,0,75,437l70.7-70.7A155.5,155.5,0,0,1,100,256Z"
+                          :style="`transform: rotate(${dialHighLimitDeg}deg)`"
+                    />
+                </svg>
+            </div>
+
+            <div class="c-dial">
+                Low Limit
+                <svg class="c-dial__bg"
+                     viewBox="0 0 512 512"
+                >
+                    <path d="M256,100c86.2,0,156,69.8,156,156s-69.8,156-156,156c-43.1,0-82.1-17.5-110.3-45.7L75,437 c46.3,46.3,110.3,75,181,75c141.4,0,256-114.6,256-256S397.4,0,256,0C185.3,0,121.3,28.7,75,75l70.7,70.7 C173.9,117.5,212.9,100,256,100z"
+                          :style="`transform: rotate(${dialLowLimitDeg}deg)`"
+                    />
+                </svg>
+            </div>
+
+            <div class="c-dial">
+                High Limit Clip 180
+                <div class="c-dial__value c-high-limit-clip--180"></div>
+            </div>
+
+            <div class="c-dial">
+                High Limit Clip 90
+                <div class="c-dial__value c-high-limit-clip--90"></div>
+            </div>
+        </div>
+    </div>
+
+</div>
+</template>
+
+<script>
+const LIMIT_PADDING_IN_PERCENT = 10;
+
+export default {
+    name: 'GaugeRadial',
+    inject: ['openmct', 'domainObject', 'composition'],
+    data() {
+        let gaugeController = this.domainObject.configuration.gaugeController;
+
+        return {
+            curVal: 0,
+            digits: 3,
+            precision: gaugeController.precision,
+            displayMinMax: gaugeController.isDisplayMinMax,
+            displayCurVal: gaugeController.isDisplayCurVal,
+            limitHigh: gaugeController.limitHigh,
+            limitLow: gaugeController.limitLow,
+            rangeHigh: gaugeController.max,
+            rangeLow: gaugeController.min,
+            gaugeType: gaugeController.gaugeType,
+            debug: false,
+            debugDial: false
+        };
+    },
+    computed: {
```

Comment:
Can we add computed properties for all the the `.indexOf` calls you're making for gauge types? ex, `isNeedle`, `isDialNeedle`, `isDial`, `isDialFilled`, etc. I think you can use this in many places in this file.